docs/sources/administration/configuration.md

```diff
@@ -1179,7 +1179,7 @@ Optional URL to send to users in notifications. If the string contains the seque
 
 ### key_file
 
-Path to JSON key file associated with a Google service account to authenticate and authorize.
+Optional path to JSON key file associated with a Google service account to authenticate and authorize. If the value is empty then it tries to retrieve authorization from GCE metadata.
```

Comment:
We could refer to application default credentials in the docs as well. Not sure if we should link to Google Cloud's docs here, but it makes sense to me.

```suggestion
Optional path to JSON key file associated with a Google service account to authenticate and authorize. If no value is provided it tries to use the [application default credentials](https://cloud.google.com/docs/authentication/production#finding_credentials_automatically).
```

Reply:
Agree with @sakjur, forgot that the docs also might need updating.